openedx/core/djangoapps/programs/tasks.py

```diff
@@ -305,6 +306,7 @@ def post_course_certificate(client, username, certificate, visible_date):
             'mode': certificate.mode,
             'type': COURSE_CERTIFICATE,
         },
+        'date_override': date_override.strftime(VISIBLE_DATE_FORMAT) if date_override else '',
```

Comment:
I don't think the Credentials PR is ready yet, but is it easier for this to be an empty string or a null? Can't remember off the top of my head which one DRF auto-handles better.

Reply:
Okay changing to `None`!